pandas/core/frame.py

```diff
@@ -1122,7 +1123,7 @@ def to_dict(self, orient='dict', into=dict):
         [{'col1': 1.0, 'col2': 0.5}, {'col1': 2.0, 'col2': 0.75}]
 
         >>> df.to_dict('index')
-        {'a': {'col1': 1.0, 'col2': 0.5}, 'b': {'col1': 2.0, 'col2': 0.75}}
+        {'a': {'col1': 1, 'col2': 0.5}, 'b': {'col1': 2, 'col2': 0.75}}
```

Comment:
@datapythonista I agree with you in general :-), but here I think the 'col1' etc actually makes it easier to see all the different possible output formats (to clearly see where the column names go in the specific output type).  
But 'a', 'b' is indeed less clear, but so I would rather call them 'row1', 'row2.

Reply:
That also makes sense. I'm personally more comfortable with something like `{'cat': {'legs': 4, 'tail': 1}}` than `{'row1': {'col1': 0.75, 'col2': 1}}`, but as far as it's not needed to spend a lot of unnecessary time until the example is understood, I'm happy with any option.

Reply:
Sorry to bother you again with the same @Moisan, but do you mind replacing the labels of the rows by `row1` and `row2`, instead of `a` and `b`? I think that will make the example clearer